renku/data/pre-commit.sh

```diff
@@ -84,17 +84,35 @@ fi
 if [ ${#ADDED_FILES[@]} -ne 0 ]; then
   UNTRACKED_PATHS=$(renku storage check-lfs-hook "${ADDED_FILES[@]}")
   if [ "$UNTRACKED_PATHS" ]; then
+
     echo 'You are trying to commit large files to Git instead of Git-LFS.'
     echo
-    echo 'Large files:'
-    for file in "${UNTRACKED_PATHS[@]}"; do
-      echo "$file"
-    done
-    echo
-    echo 'To track these files in Git LFS use "git lfs track <FILENAMES>".'
-    echo
-    echo 'To commit anyway, use "git commit --no-verify".'
-    exit 1
+    AUTOCOMMIT_LFS=$(renku config show autocommit_lfs)
```

Comment:
This should allow us to set it outside of the project configuration so we don't need to make changes to the user's project in the autosave hook. E.g.

```
AUTOCOMMIT_LFS=${AUTOCOMMIT_LFS:=$(renku config show autocommit_lfs)}
```

I _think_ that should work but I'm not sure...